examples/mongodbatlas_federated_settings_identity_provider/azure/README.md

```diff
@@ -0,0 +1,231 @@
+# MongoDB Atlas Provider -- OIDC Workload Azure
+
+## Dependencies
+
+- Terraform [MongoDB Atlas Provider v1.17.0](https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs)
+- Terraform [Hashicorp Cloud Init Provider v2.3.4](https://registry.terraform.io/providers/hashicorp/cloudinit/latest/docs)
+- Terraform [Hashicorp Azurerm Provider v3.106.1](https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs)
+- Terraform [Hashicorp Random Provider v3.6.2](https://registry.terraform.io/providers/hashicorp/random/latest/docs)
+- A MongoDB Atlas account with an Organization configured with [Federation Authentication](https://www.mongodb.com/docs/atlas/security/federated-authentication/#federation-management-console)
+  - Get the `federated_settings_id` from the url, e.g., <https://cloud.mongodb.com/v2#/federation/{federated_settings_id}/overview>
```

Comment:
can also be retrieved through `mongodbatlas_organization` resource or data source https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/resources/organization#argument-reference